### DISCUSSION TITLE
Search functionality

### DISCUSSION THREAD

**Discussion by RexRexRexx** [Taewan-P/gpt_mobile#140] [Ideas]
I've been using the GPTMobile app a lot, and with many chats piling up, it's become difficult to find a specific conversation from the past that I know I had. A search bar would be super helpful in this case, allowing me to quickly access previous chats and recall valuable info.

**Comment by nanderer** [comment]
yea full text search through all sessions with maybe an integration in the android sytemwide search, is that btw still a thing, i last saw it on 2.3 ;)

**Comment by Taewan-P** [comment]
Hello, this feature is now implemented in [v0.7.0](https://github.com/Taewan-P/gpt_mobile/releases). Thanks!
